api/hypershift/v1beta1/hosted_controlplane.go

```diff
@@ -302,6 +302,18 @@ type HostedControlPlaneStatus struct {
 	// +optional
 	ReleaseImage string `json:"releaseImage,omitempty"`
 
+	// AvailableUpdateVersions is a comma separated list of all available update versions.
+	//
+	// Deprecated: Use versionStatus.availableUpdates instead.
+	// +optional
+	AvailableUpdateVersions string `json:"availableUpdateVersions,omitempty"`
+
+	// ConditionalUpdateVersions is a comma separated list of all conditional update versions.
+	//
+	// Deprecated: Use versionStatus.conditionalUpdates instead.
+	// +optional
+	ConditionalUpdateVersions string `json:"conditionalUpdateVersions,omitempty"`
```

Comment:
Conditional update version strings are probably not all that useful without knowing which of the risks associated with each target the cluster-version operator was concerned about.  Maybe just focus on `availableUpdates` for this hopefully-short-term workaround?